### ISSUE TITLE
Inviting people to contribute in this project.

### ISSUE THREAD

**Issue by zainulabidin302** [zainulabidin302/laravel-project-001#2]
How can we invite people to contribute to this project?

Particularly those people who knows Laravel and Vue-js very well.

**Comment by anasmajeed** [comment]
Let us make a working model first and then invite people so at least they can see something... What do you say?

**Comment by zainulabidin302** [comment]
I was thinking we need some people who can comment on the UI/UX.

**Comment by anasmajeed** [comment]
Yup but still we have no working module to show what if basic functionality is working and then we show someone so they can get good idea of what is going on... lets meet and done it and then add people to see
